eng/Versions.props

```diff
@@ -7,13 +7,13 @@
 <Project>
   <PropertyGroup Label="Version settings">
     <AspNetCoreMajorVersion>3</AspNetCoreMajorVersion>
-    <AspNetCoreMinorVersion>1</AspNetCoreMinorVersion>
-    <AspNetCorePatchVersion>2</AspNetCorePatchVersion>
-    <PreReleasePreviewNumber>0</PreReleasePreviewNumber>
+    <AspNetCoreMinorVersion>2</AspNetCoreMinorVersion>
+    <AspNetCorePatchVersion>0</AspNetCorePatchVersion>
+    <PreReleasePreviewNumber>1</PreReleasePreviewNumber>
```

Comment:
Should this be 2 for `Preview2`?

Reply:
Up to you if you want me to bump this here, or if you want to bump it in a separate PR

Reply:
```suggestion
    <PreReleasePreviewNumber>2</PreReleasePreviewNumber>
```